xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/XWiki.java

```diff
@@ -1042,9 +1042,8 @@ public static String getServerWikiPage(String servername)
 
     /**
      * @param content the content of the text area
-     * @param context see {@link XWikiContext}
      */
-    public static String getTextArea(String content, XWikiContext context)
+    public static String getTextArea(String content)
```

Comment:
Actually you can't do that in this manner since you'd be breaking an API :)

You'll need to introduce new method, deprecate the old one (and move it to the legacy module). See http://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HBackwardCompatibility